### PR TITLE
Add note about git hooks

### LINKED ISSUE #16: Git hooks
Every once in a while it happens that I forget to run `lint` or `coverage` or some command before pushing, and only realize there was an error through CI. I will have to define git hooks locally for myself, but I was thinking maybe this can also be added to the projects so that automatically before push or before commit at least linting would run. [Husky](https://github.com/typicode/husky) seems interesting and easy to integrate. What do you all think?

### PR DESCRIPTION
**Pull request by evertonfraga**
Closes #16.

### CODE REVIEW

**Review by holgerd77 — APPROVED** [review]
LGTM

File: packages/lint/README.md [hunk 1]
```diff
@@ -37,5 +37,9 @@ Use CLI commands above in your `package.json`:
   }
 ```
 
+## Taking more of linting
```
Comment (ryanio) [inline packages/lint/README.md:40]: taking?

Comment (holgerd77) [reply packages/lint/README.md:40]: Ah, also couldn't read that but assumed it was some kind of English slang I don't understand, lol 😜

Comment (evertonfraga) [reply packages/lint/README.md:40]: sorry! I did literal translation. 

Fix here: https://github.com/ethereumjs/ethereumjs-config/blob/master/packages/lint/README.md

**Pull request merged**